config.ts

```diff
@@ -45,6 +55,12 @@ export interface Pool extends BasePool {
     SENIOR_OPERATOR: 'ALLOWANCE_OPERATOR' | 'PROPORTIONAL_OPERATOR'
     partialRepay?: boolean
   }
+  archivedValues?: {
```

Comment:
I think these don't need to be here right, since this is the type for active pools?